File: teste.py
#Criando uma classe Cachorro
class Cachorro:
    def __init__(self, nome, raca):
        self.nome = nome
        self.raca = raca

    def latir(self):
        print(f"{self.nome} está latindo!")

#Criando dois objetos a partir da classe Cachorro
cachorro1 = Cachorro("Rex", "Labrador")
cachorro2 = Cachorro("Bella", "Poodle")

#Chamando o método latir para cada objeto
cachorro1.latir() #saída: Rex está latindo!
cachorro2.latir() #saída: Bella está latindo!

File: teste.java
public class Pessoa 
{
    String nome;

    // Constructor
    public Pessoa(String nome) {
        this.nome = nome;
    }

    //Método
    public void apresentar() {
        System.out.println(
            "Olá, meu nome é " + this.nome
        );
    }
}

public class Main {
    public static void main(String[] args) {
        Pessoa p1 = new Pessoa("João");
        p1.apresentar();
    }
}


